react-design/react-design/src/PreviousButton.tsx

```diff
@@ -0,0 +1,6 @@
+type Props = {
+  onPrev: () => void;
```

Comment:
If you were to pass in the text, the NextButton and PreviousButton could be the same component.